tests/other_test.go

```diff
@@ -22,3 +24,44 @@ func TestRunRestoredDBExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunPointInTimeRecoveryDBExample(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: pitrTerraformDir,
+		Prefix:       "pg-pitr",
+		TerraformVars: map[string]interface{}{
+			"pitr_id": permanentResources["postgresqlCrn"],
+			"region":  region,
+		},
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func testRunBasicExample(t *testing.T, version string) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       basicExampleTerraformDir,
+		Prefix:             "postgres",
+		BestRegionYAMLPath: regionSelectionPath,
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunBasicExample(t *testing.T) {
+	t.Parallel()
+	versions := []string{"12", "13"} // Always test for version which are already not included in the complete and FSCloud tests.
```

Comment:
Since we actually already have two other examples that we could test these versions with (PITR and restore) then lets lock those into these versions, and pass no version into the basic example so it will always use the current ICD default version

Reply:
The restore test would create 4 Postgres instances (primary and backup * 2) rather than just 2 if we test against 2 different versions. I think we can keep as is because the purpose of having different versions tested is to make sure about things are working fine - that can be tested in the basic example as well.